sktime/forecasting/fbprophet.py

```diff
@@ -111,6 +111,10 @@ class Prophet(_ProphetAdapter):
     stan_backend: str or None, default=None
         str as defined in StanBackendEnum. If None, will try to
         iterate over all available backends and find the working one.
+    fit_kwargs: dict or None, default=None
```

Comment:
It would really be better to give a reference or link or list of possible parameters, or this will not be useful except for users who know prophet under the hood. Not blocking though.

Reply:
@fkiraly 
Unfortunately Prophet itself does not provide this list of possible parameters anywhere in their documentation or source code. It's up to you ofc but I don't think we should make this sktime's responsibility to document. Also, note again that the accepted parameters will depend on the Stan backend being used. And when taking the default `cmdstanpy` backend, there are more than 30 possible parameters... 😬 (see [here](https://mc-stan.org/cmdstanpy/api.html#cmdstanpy.CmdStanModel.optimize) and [here](https://mc-stan.org/cmdstanpy/api.html#cmdstanpy.CmdStanModel.sample))

Reply:
@tpvasconcelos, I would suggest you add a short version of this to the docstring then, precisely what you said - prophet does not document this, our best guess is the stan backend params, link and link. Slightly more docstring tone.

Will merge this for now for the release, we can look at the docstring separately.